### ISSUE TITLE
Applying weights to loss

### ISSUE THREAD

**Issue by rowem5** [paninski-lab/lightning-action#16]
Currently weights are appropriately calculated and logged to the config but they are not applied to the cross entropy loss function. This fix would help  in addressing class imbalance.
